pkg/cloudprovider/provider/anexia/provider.go

```diff
@@ -121,14 +121,14 @@ func (p *provider) Create(ctx context.Context, log *zap.SugaredLogger, machine *
 	}()
 
 	// provision machine
-	err = provisionVM(ctx, log, client)
+	err = provisionVM(ctx, log, client, providerCfg)
 	if err != nil {
 		return nil, anexiaErrorToTerminalError(err, "failed waiting for vm provisioning")
 	}
 	return p.Get(ctx, log, machine, data)
 }
 
-func provisionVM(ctx context.Context, log *zap.SugaredLogger, client anxclient.Client) error {
+func provisionVM(ctx context.Context, log *zap.SugaredLogger, client anxclient.Client, providerCfg *providerconfigtypes.Config) error {
```

Comment:
What's your opinion on adding the providerCfg to the reconcileContext?
Would reduce the amount of parameters.. But not sure if we"d gain anything from it...

Reply:
Good point, moved to `reconcileContext`